src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -1157,10 +1165,32 @@ std::vector<ModifierSum::modifier_entry_t> CountryInstance::get_contributing_mod
 	return modifier_sum.get_contributing_modifiers(effect);
 }
 
-void CountryInstance::update_gamestate(
-	DefineManager const& define_manager, UnitTypeManager const& unit_type_manager,
-	ModifierEffectCache const& modifier_effect_cache
-) {
+void CountryInstance::update_gamestate(InstanceManager& instance_manager) {
+	owns_colonial_province = std::any_of(
+		owned_provinces.begin(), owned_provinces.end(),
+		[](ProvinceInstance const* province) -> bool {
+			using enum ProvinceInstance::colony_status_t;
```

Comment:
It's shorter, clearer and easier to read to remove this line and just type `ProvinceInstance::colony_status_t::STATE`